packages/firestore/src/api/database.ts

```diff
@@ -398,14 +398,30 @@ export class Firestore implements firestore.FirebaseFirestore, FirebaseService {
           'any other methods on a Firestore object.'
       );
     }
+
+    let synchronizeTabs = false;
+
+    if (settings) {
+      if (settings.experimentalTabSynchronization !== undefined) {
+        log.error(
+          'The `experimentalTabSynchronization` setting has been renamed to ' +
```

Comment:
This is probably super nitty but I don't think we usually use backticks within log messages.  I'd remove or replace with `'`

Reply:
I didn't like the message without any quoting, but I am fine just using single quotes.